ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/TraceCallMany.java

```diff
@@ -0,0 +1,180 @@
+/*
+ * Copyright Hyperledger Besu.
```

Comment:
nit: Copyright contributors to Hyperledger Besu

Reply:
[c4f0086](https://github.com/hyperledger/besu/pull/3468/commits/c4f008654f1582a71db15a3d2d18de30fed3949b)